lifecycle-operator/apis/options/v1alpha1/keptnconfig_types.go

```diff
@@ -47,6 +47,15 @@ type KeptnConfigSpec struct {
 	// +kubebuilder:default:=true
 	// +optional
 	BlockDeployment bool `json:"blockDeployment,omitempty"`
+
+	// ObservabilityTimeout specifies the maximum time to observe the deployment phase of KeptnWorkload.
+	// If the workload does not deploy successfully within this time frame, it will be
+	// considered as failed.
+	// +kubebuilder:default:="5m"
+	// +kubebuilder:validation:Pattern="^0|([0-9]+(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))+$"
```

Comment:
Are we sure about `(ns|us|µs|ms|s|m|h))+`? 
the last `+` would allow `5m5m5m` 🤔 
I would expect the following
```suggestion
	// +kubebuilder:validation:Pattern="^0|([0-9]+(\\.[0-9]+)?(ns|us|µs|ms|s|m|h))$"
```

Reply:
I would say it's expected, as we want to support format such as `1h30m`, no?

Reply:
you are right about `5m5m5m`, but I think the duration data type can handle also this input properly, so basically it's valid (although not ideal I agree)

Reply:
looking around Format=duration is not an option, and this regex seems to be the winner used also in apimachinery ex.  see here the openshift pr https://github.com/openshift/hive/pull/1690/files